daml-lf/spec/transaction.rst

```diff
@@ -232,12 +230,7 @@ As of version 10, these fields are included:
 
 ``template_id`` and ``value`` are required; ``agreement`` is optional.
 
-``value`` must conform to the type of the Daml-LF associated data type
-indicated by ``template_id``.
-
-``template_id``'s structure is defined by `the value specification`_;
-the version of that specification to use when consuming it is the
-``version`` field of ``value``.
```

Comment:
I dropped everything that is relative to type. This is handle by type checking not (de)serialization processes.

Reply:
what happened to the version field of value? Don’t we still need that for version 10?

Reply:
`the version of that specification to use when consuming it is the ``version`` field of ``value``.`

This was relevant for legacy value versions that have been dropped